ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
+%LUA_CPATH%\lua5.1 -v || goto :exit
+
+cd tests
+REM Each suite should run individually
+REM Add them here
+@echo on
+%LUA_CPATH%\lua5.1 core_suite.lua
+@echo off
+cd ..
+@echo on
+:exit
```

Comment:
> REM Test that lua is installed
>  %LUA_CPATH%\lua5.1 -v || (echo "Unable to run Lua executable" && exit)
Works good. You don't really need that message. You'd get something like 
> D:\Games\minetest-0.4.14-server\mods\ctf_pvp_engine\ctf>check
> "c:\lua\\lua25.1" error: unable to execute command: program not executable
> "Unable to run Lua executable"